Remove the prefix when finding the gdk_picbuf_query_loader helper program.
This allowed the librsvg Gdk plugin to be built and installed.

author: micketeer@gmail.com
--- gdk-pixbuf-loader/meson.build.orig	2025-12-20 18:49:30.253566655 +0100
+++ gdk-pixbuf-loader/meson.build	2025-12-20 19:06:02.798622117 +0100
@@ -43,7 +43,7 @@
 pixbufloader_svg_install_args = [
   '--gdk-pixbuf-moduledir',
   pixbuf_dep.get_variable(pkgconfig: 'gdk_pixbuf_moduledir', pkgconfig_define: ['prefix', prefix]),
-  pixbuf_dep.get_variable(pkgconfig: 'gdk_pixbuf_query_loaders', pkgconfig_define: ['prefix', prefix]),
+  pixbuf_dep.get_variable(pkgconfig: 'gdk_pixbuf_query_loaders'),
   pixbuf_dep.get_variable(pkgconfig: 'gdk_pixbuf_cache_file', pkgconfig_define: ['prefix', prefix])
 ]
 
